src/VisualStudio/CSharp/Test/CSharpVisualStudioTest.csproj

```diff
@@ -103,8 +103,9 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
     <ProjectReference Include="..\..\..\Features\Core\Portable\Features.csproj">
-      <Project>{edc68a0e-c68d-4a74-91b7-bf38ec909888}</Project>
+      <Project>{EDC68A0E-C68D-4A74-91B7-BF38EC909888}</Project>
       <Name>Features</Name>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
Actually, I looked at the rest of this project, and we do this for many of the other references.  My guess is we shouldn't have deleted this in the first place.  We should just make sure we don't delete it again in features-testimpact.